### PR TITLE
Checkpoint pull request

### PR DESCRIPTION
**Pull request by helljack26**
Hi Vlad 
In this homework I have added Api films render to main page, a little styles and displaying genres of film . 
Doesn't work query input, detail film info and watch list with localStorage saving.

### CODE REVIEW

File: src/components/FilmCard/FilmCard.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { Link } from 'react-router-dom';
+import style from './FilmCard.module.css';
+const FilmCard = ( { name, image, genre, id } ) =>
+{
+  const arr = genre.map( ( genre, id ) => id !== genre.length - 1 ? <span className={style.genre} key={id}>{genresConvert( genre )}, </span> : <span className={style.genre} key={id}>{genresConvert( genre )}.</span> )
```
Comment (Oodmincheg) [inline src/components/FilmCard/FilmCard.jsx:5]: Arr is not very good naming :) `genres` it's genres or what?

File: src/components/FilmCard/FilmCard.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      <img src={`https://image.tmdb.org/t/p/w500/${image}`} alt="film poster" />
+      <div>Жанр: {arr}</div>
+    </Link>
+  );
+};
+
+export default FilmCard;
+
+// Convert api code genres to russian definition
+function genresConvert( genre )
```
Comment (Oodmincheg) [inline src/components/FilmCard/FilmCard.jsx:18]: Firstly you can create just object with key and value. Secondly you can just `return value` and not to do `genreContent = value; break` everywhere. And move this function to `./helpers` file in the component folder.

File: src/components/FilmList/FilmList.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import FilmCard from '../FilmCard';
+import QueryInput from '../QueryInput';
+import style from './FilmList.module.css'
+
+const FilmList = () =>
+{
+  const [film, setFilms] = useState( [] );
+  const [gender, setGender] = useState( '' );
```
Comment (Oodmincheg) [inline src/components/FilmList/FilmList.jsx:10]: Gender?

File: src/components/FilmCard/FilmCard.jsx (hunk 2 above)
Comment (helljack26) [reply src/components/FilmCard/FilmCard.jsx:18]: > Firstly you can create just object with key and value. Secondly you can just `return value` and not to do `genreContent = value; break` everywhere. And move this function to `./helpers` file in the component folder.

Good idea, thank you

File: src/components/FilmList/FilmList.jsx (hunk 3 above)
Comment (helljack26) [reply src/components/FilmList/FilmList.jsx:10]: Removed this gender)

**Pull request merged**